interactives/unicode-length/js/unicode-length.js

```diff
@@ -0,0 +1,8 @@
+$(document).ready(function () {
+  $("#interactive-unicode-length-button").on('click', function() {
+    var text = $("#interactive-unicode-length-text").val();
+    $("#interactive-unicode-length-utf8").html(bitLength(text, 8) + " bits");
+    $("#interactive-unicode-length-utf16").html(bitLength(text, 16) + " bits");
+    $("#interactive-unicode-length-utf32").html(text.length * 32 + " bits");
```

Comment:
Perhaps this should be included in the unicode.js bitLength function too? I know it's trivial, but that way the logic is kept together. 

Not too fussed either way, I'll leave it up to you :)

Reply:
This is specific to this interactive, while `bitLength` was separated out as I'll import it for other Unicode interactives.